operator/pkg/clusterconfiguration/expander.go

```diff
@@ -11,12 +11,14 @@ import (
 	"strconv"
 	"strings"
 
+	"emperror.dev/errors"
```

Comment:
```suggestion
	"github.com/cockroachdb/errors"
```

we are trying to use only cockroachdb/errors package. Could you change that import?